lrpar/examples/mut_param/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "mut_param"
```

Comment:
We could call this `clone_param` maybe?

Reply:
Should be resolved in [bf4ada3](https://github.com/softdevteam/grmtools/pull/471/commits/bf4ada313fdf1a78b517d5d872f5a563718ff57e).